### DISCUSSION TITLE
Providing documentation for contributers.

### DISCUSSION THREAD

**Discussion by OurCodeBase** [termux-user-repository/tur#645] [General]
Termux is now going to almost dead, there are few recent commits in repo. There are almost no contributers are working on its official package repo.  As I see this project is a great opportunity for termux users to install new and updated packages. So I'm Interested to add new packages here, but I don't have any Idea. Can you please provide documentation that how can I add new packages into tur repo ? So I will be a regular contributor and maintainer of this project.

**Comment by licy183** [comment]
Ah, contributions are welcome. The steps is the same as the Termux main repository, since TUR uses the same build system as the main repository.

**Comment by knyipab** [comment]
I revised the Contribution section of README.md few days ago, with instructions and links to official wiki. Along with the use of dev container, contribution to TUR should not be difficult.
